components/cards/CardProgram.tsx

```diff
@@ -2,41 +2,49 @@ import stls from '@/styles/components/cards/CardProgram.module.sass'
 import Link from 'next/link'
 import Price from '@/components/costs/Price'
 import { IconArrowTopRight } from '@/components/icons'
-import { getStringDeclensionNumber } from '@/helpers/index'
-import { SetString } from '@/helpers/index'
+import { getStringDeclensionNumber, SetString } from '@/helpers/index'
 import lang from '@/data/translation/months'
+import TrainingPeriod from '@/components/costs/TrainingPeriod'
+import useAt from '@/helpers/useAt'
 
 const CardProgram = ({ professionLayout, program, number, type, format }) => {
+  const at = useAt()
+
   return (
     <Link href={`/programs/${type}/${format}/${program.slug}`}>
       <a className={stls.container}>
         <div className={stls.arrow}>
           <IconArrowTopRight />
         </div>
-        <div className={stls.number}>
-          {number < 10 ? `0${number}` : number}.
+        <div>
+          <span className={stls.category}>
+              {at.mini ? 'mini MBA' : at.mba ? 'MBA' : at.profession ? 'Профессии' : at.course ? 'Курсы' : ''}
```

Comment:
labels should be singular
```suggestion
 {at.mini ? 'Mini MBA' : at.mba ? 'MBA' : at.profession ? 'Профессия' : at.course ? 'Курсы' : 'Курс'}
```